frame/staking/remote-test/src/lib.rs

```diff
@@ -0,0 +1,307 @@
+#![cfg(test)]
```

Comment:
I am 95% sure that I would prefer removing this code before we merge. In case of the 5%: 

- [ ] needs license
- [ ] needs to be `#[ignore]`ed. 

To play around with this, clone `remote-externalities ` next to your substrate folder, and make all of its dependencies point to your local substrate. [You can do that with `node update_cargo.js local`](https://github.com/paritytech/substrate-debug-kit/blob/master/update_cargo.js)

Reply:
removed.